src/main/java/com/powsybl/openloadflow/sa/DcSecurityAnalysis.java

```diff
@@ -251,11 +251,6 @@ private List<OperatorStrategyResult> createOperatorStrategyResults(DcSecurityAna
                 .setCacheEnabled(false); // force not caching as not supported in secu analysis
 
         try (LfNetworkList lfNetworks = Networks.load(network, dcParameters.getNetworkParameters(), topoConfig, Reporter.NO_OP)) {
-
```

Comment:
I am worried about your change using `LoadFlowParameters` here, and for DC sensitivity analysis. Because we force for these use cases no shunt voltage control and no HVDC AC emulation. But here you can configure `true` without any issue for a DC load flow. But for the creation of the propagated contingency, it will be wrong.